keel-actuator/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuator.kt

```diff
@@ -101,6 +103,9 @@ class ResourceActuator(
           }
         }
       }
+    } catch (e: ResourceDependencyNotFound) {
+      log.warn("Resource check for {} failed (hopefully temporarily) due to {}", id, e.message)
+      publisher.publishEvent(ResourceCheckDependencyMissing(resource, e, clock))
```

Comment:
I read this as "the resource check dependency is missing". Maybe rename to `ResourceCheckMissingDependency`?